misc/scripts/install-local.sh

```diff
@@ -483,24 +472,23 @@ if [[ $REMOVE_DEPENDS = y ]]; then
 	sudo apt-get remove $build_depends
 fi
 
-cd "$HOME"
+cd "$HOME" || exit 1
 
 # Metadata writing
 log
 
 fancy_message info "Symlinking files"
-cd /usr/src/pacstall/ || sudo mkdir -p /usr/src/pacstall && cd /usr/src/pacstall
+cd /usr/src/pacstall/ || sudo mkdir -p /usr/src/pacstall && cd /usr/src/pacstall || exit 1
 # By default (I think), stow symlinks to the directory behind it (..), but we want to symlink to /, or in other words, symlink files from pkg/usr to /usr
 if ! command -v stow > /dev/null; then
 	# If stow failed to install, install it
 	sudo apt-get install stow -y
-	cd /usr/src/pacstall
+	cd /usr/src/pacstall || exit 1
```

Comment:
`{ fancy_message error "/usr/src/pacstall does not exist" && exit 1 }`

Reply:
Fixed in https://github.com/pacstall/pacstall/pull/276/commits/d7e37217811b08651b372e9e47cb6ce6b6357858